cfpb/cfgov-refresh
# Adding safe filter to render_block macro

Adding `safe` filter to render_block macro to fix issue causing the Careers Jobs Listing table to display as text.

## Changes

- Adding safe filter to render_block macro

## Testing

- Add a Browse Page via Wagtail.
- Add a `Jobs Listing Table` to the browse page.
- Preview and it verify it displays as expected.

## Review

- @chosak 

## Screenshots

Before:

<img width="849" alt="screen shot 2016-09-21 at 3 25 21 pm" src="https://cloud.githubusercontent.com/assets/1696212/18725787/a454c1e6-800f-11e6-9a04-81445d01f14a.png">

After:

<img width="531" alt="screen shot 2016-09-21 at 3 24 40 pm" src="https://cloud.githubusercontent.com/assets/1696212/18725769/90c21994-800f-11e6-9cc8-981eb4da8ed5.png">


## Notes

- I tried using `MarkSafe` and `mark_safe` to no avail.


## Checklist
